lib/ansible/modules/copy.py

```diff
@@ -670,13 +680,14 @@ def main():
                     _, b_mysrc = tempfile.mkstemp(dir=os.path.dirname(b_dest))
 
                     shutil.copyfile(b_src, b_mysrc)
-                    try:
-                        shutil.copystat(b_src, b_mysrc)
-                    except OSError as err:
-                        if err.errno == errno.ENOSYS and mode == "preserve":
-                            module.warn("Unable to copy stats {0}".format(to_native(b_src)))
-                        else:
-                            raise
+                    if not inhibit_copystat:
```

Comment:
i would still not make this an option, but try to capture the copystat error on hp/ux and issue the warning as we do for enosys case

Reply:
> i would still not make this an option, but try to capture the copystat error on hp/ux and issue the warning as we do for enosys case

This is not HP/UX, but HPE NonStop. There is no specific error code for this situation and the error message varies depending on circumstance - I think `LANG` might be involved too, so parsing the stderr output may be impractical. The general error - not specific to this situation - is `Invalid function argument`, which can result from many causes but is part of the message here of significance at least with the `cp` command, attempting to copy k from one directory to another in the edge condition described in this PR:
```
cp: preserving permissions for '/G/data01/randall/k': Invalid function argument
```

Reply:
LANG should not be involved as we are making a system call, so answers should always be in POSIX/C locale, even if we were using the command line tools we do have the ability to force the locale/lang setting to enable parsing the output.